src/auro-input.js

```diff
@@ -281,14 +283,15 @@ export default class AuroInput extends BaseInput {
           <p class="inputElement-helpText" id="${this.uniqueId}">${this.helpText}</p>
         ` : html`
           <p class="inputElement-helpText error" id="${this.uniqueId}" role="alert" aria-live="assertive">${this.getErrorMessage()}</p>
-        `}
-        ${this.type === 'credit-card' ? this.processCreditCard() : undefined}
-        ${this.type === 'credit-card' && this.icon ? html`<auro-icon class="creditCard-icon" category="payment" name="credit-card"></auro-icon>` : undefined}
-        ${this.inputIconName === 'cc-amex' ? html`<auro-icon class="creditCard-icon" category="payment" name="cc-amex"></auro-icon>` : undefined}
-        ${this.inputIconName === 'cc-visa' ? html`<auro-icon class="creditCard-icon" category="payment" name="cc-visa"></auro-icon>` : undefined}
-        ${this.inputIconName === 'cc-mastercard' ? html`<auro-icon class="creditCard-icon" category="payment" name="cc-mastercard"></auro-icon>` : undefined}
-        ${this.inputIconName === 'cc-discover' ? html`<auro-icon class="creditCard-icon" category="payment" name="cc-discover"></auro-icon>` : undefined}
-        ${this.inputIconName === 'cc-alaska' ? html`<auro-icon class="creditCard-icon" category="payment" name="cc-alaska"></auro-icon>` : undefined}
+        `
+      }
+      ${this.type === 'credit-card' ? this.processCreditCard() : undefined}
+      ${this.inputIconName ?
+        repeat([this.inputIconName], x => x, (name) =>
```

Comment:
I think it's worth a comment here detailing what we're trying to accomplish.

Reply:
100. Even though I am approving this update to make this code less cumbersome, I don't want to lose sight of the real issue that auro-icon isn't updating the fetch call with the attribute update. https://github.com/AlaskaAirlines/auro-icon/issues/31